app/controllers/operating_systems_controller.rb

```diff
@@ -19,39 +18,28 @@ def new
   def edit; end
 
   # POST /operating_systems
-  # POST /operating_systems.json
   def create
     @operating_system = OperatingSystem.new(operating_system_params)
-
-    respond_to do |format|
-      if @operating_system.save
-        format.html { redirect_to operating_systems_url, notice: 'Operating system was successfully created.' }
-      else
-        format.html { render :new }
-      end
+    if @operating_system.save
+      redirect_to operating_systems_url, notice: 'Operating system was successfully created.'
+    else
+      render :new
     end
   end
 
   # PATCH/PUT /operating_systems/1
-  # PATCH/PUT /operating_systems/1.json
   def update
-    respond_to do |format|
-      if @operating_system.update(operating_system_params)
-        format.html { redirect_to operating_systems_url, notice: 'Operating system was successfully updated.' }
-      else
-        format.html { render :edit }
-      end
+    if @operating_system.update(operating_system_params)
+      redirect_to operating_systems_url, notice: 'Operating system was successfully updated.'
+    else
+      render :edit
     end
   end
 
   # DELETE /operating_systems/1
-  # DELETE /operating_systems/1.json
   def destroy
     @operating_system.destroy
-    respond_to do |format|
-      format.html { redirect_to operating_systems_url, notice: 'Operating system was successfully destroyed.' }
-      format.json { head :no_content }
-    end
+    redirect_to operating_systems_url, notice: 'Operating system was successfully destroyed.'
```

Comment:
Could we better use `deleted` instead of `destroyed` here (and in `app/controllers/request_templates_controller.rb` line 42)? `Destroyed` always sounds weird for me.

Reply:
Good point. Anyways, this should be standardized using a single string in I18n, which is used in all controllers.